libraries/AP_Baro/examples/ICM20789/ICM20789.cpp

```diff
@@ -35,6 +35,9 @@ static AP_HAL::OwnPtr<AP_HAL::Device> dev;
 #define CMD_SOFT_RESET     0x805D
 #define CMD_READ_ID        0xEFC8
 
+void setup(void);
+void loop(void);
```

Comment:
Where are you seeing a warning with these two? The only warnings I saw with this example were:

```
../../libraries/AP_Baro/examples/ICM20789/ICM20789.cpp:41:13: warning: ‘void spi_init()’ defined but not used [-Wunused-function]
 static void spi_init()
             ^
../../libraries/AP_Baro/examples/ICM20789/ICM20789.cpp:219:13: warning: ‘void i2c_init()’ defined but not used [-Wunused-function]
 static void i2c_init(void)
```

Which will continue to happen.

Reply:
![image](https://user-images.githubusercontent.com/7996972/37294673-0a309a24-261f-11e8-8569-66977339281c.png)

Reply:
Warnings are from Travis px4.